.circleci/config.yml

```diff
@@ -1,6 +1,14 @@
-version: 2
+version: 2.1
+
 jobs:
   build:
+    parameters:
+      docker_image:
+        type: string
+
+    parallelism: 2
+
+    # Need docker run with SYS_PTRACE for gcc/Asan address sanitizer to work
```

Comment:
This is confusing. We don't run with Address Sanitizer in gcc, we use clang for that purpose.